PWGHF/TableProducer/HFCandidateCreator3Prong.cxx

```diff
@@ -193,7 +193,7 @@ struct HFCandidateCreator3ProngExpressions {
 
           //Printf("Flagging the different Λc± → p± K∓ π± decay channels");
           if (arrayDaughters[0].has_mcParticle()) {
-            swapping = int8_t(std::abs(arrayDaughters[0].mcParticle_as<aod::McParticles_000>().pdgCode()) == kPiPlus);
+            swapping = int8_t(std::abs(arrayDaughters[0].mcParticle_as<aod::McParticles>().pdgCode()) == kPiPlus);
```

Comment:
`mcParticle()`

Reply:
Done! (everywhere)

Reply:
`arrayDaughters[0].mcParticle().pdgCode()` did not work here?